modules/invibesBidAdapter.js

```diff
@@ -635,10 +629,29 @@ function readGdprConsent(gdprConsent) {
     }
 
     return 2;
+  } else {
+    invibes.GdprModuleInstalled = false;
+    invibes.UspModuleInstalled = !!usConsent;
+    if (usConsent?.length && usConsent.length > 2 && usConsent[2] == 'N') {
+      setAllPurposesAndLegitimateInterests(true);
+      return 2;
+    }
+
+    setAllPurposesAndLegitimateInterests(false);
+    invibes.GdprModuleInstalled = false;
```

Comment:
It looks like a duplicate of [L.633](https://github.com/prebid/Prebid.js/pull/12183/files#diff-a10e7674ac5088c0a6a97e46ebaf70ab91660cb5acbe8669132f897747f399f6R633)

Reply:
It seems your changes broke the tests, can you check ?
https://app.circleci.com/pipelines/github/prebid/Prebid.js/22765/workflows/384b44d5-4db5-480b-b4d9-ad1b84b00cdc/jobs/40312?invite=true#step-106-80426_20